cassandra/roles/install-cassandra/tasks/main.yml

```diff
@@ -0,0 +1,19 @@
+---
+- name: Create a namespace for cassandra
+  k8s:
+    name: "{{ cassandra_namespace }}"
+    api_version: v1
+    kind: Namespace
+    state: present
+- name: Switch to the created namespace
```

Comment:
You believe you can remove this task and instead specify the namespace in the `oc adm` command:
`oc adm policy add-scc-to-user privileged -z default -n {{cassandra_namespace}}`

I'm fine with merging this for now but we will also want to revisit exactly what this above command is doing. I will open up an issue to explore the security requirements of Cassandra and whether or not we can modify what we are doing here.

Reply:
i rwill remove the task to switch to the cassandra namespace and specify it in the namespace using the command you gave. does {{cassandra_namespace}} need to be in quotes? i tested it out without the quotes and it seems fine but just wanted to confirm.